showpass/Showpass-E2E-tests
# TA-63 Cypress cloud config

**Description:**
With the Cypress cloud analytics we will be able to better understand test results statistics and easily debug failing tests. Video recordings and screenshots will be available in the cloud.
Adding the following projectId to connect our scripts with Cypress cloud projectId: ‘e9axi7'

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-63

**How to test:**
Run the following command on your local to see the results recorded in the cloud
`npx cypress run --record --key 87930fca-2207-4d65-9f89-2c86dd67c691`

**Test execution results:**

<img width="919" alt="TA-63-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/ad67a07a-7af6-4c2d-8202-47fe78c55c00">
